src/main/java/svc/data/citations/filters/FilterCitations.java

```diff
@@ -0,0 +1,35 @@
+package svc.data.citations.filters;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.ListIterator;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+import svc.managers.CourtManager;
+import svc.models.Citation;
+import svc.models.Court;
+
+@Component
+public class FilterCitations {
```

Comment:
Generally speaking, class names should be things, not actions; i.e. CitationFilter or CitationExpirationFilter.